app/views/shared/_taxonomy_navigation.html.erb

```diff
@@ -17,7 +17,11 @@
 
         <%= render "govuk_publishing_components/components/highlight_boxes", {
           inverse: true,
-          items: taxonomy_navigation[:services]
+          items: taxonomy_navigation[:services][:promoted_content]
+        } %>
+
+        <%= render "govuk_publishing_components/components/document_list", {
```

Comment:
One of the comments from the accessibility review was that the things in the highlight boxes and the document list should all be in one html list. I'm not sure how we'd do that with the existing components.  It might be worth us having a chat with the team to see what the best way to resolve this is.

Reply:
I think it's ok to go ahead with this for now as we currently have this problem in a few places (e.g: topic pages) too. I've added a card in the backlog to remind us to revisit the problem, but I'm also not sure how it can work with our existing components as they need to be able to be reused without relying on code being in place. https://trello.com/c/BLsoPJJk/100-spike-can-we-put-highlight-boxes-and-document-list-items-in-one-list